src/client/pydaos/raw/daos_api.py

```diff
@@ -34,6 +34,7 @@
 
 from . import daos_cref
 from . import conversion
+from .. import DaosClient
```

Comment:
(pylint-import-error) Unable to import 'pydaos' (inconsistent use of tabs and spaces in indentation (<string>, line 73))